pkgs/applications/virtualization/docker/default.nix

```diff
@@ -222,5 +227,6 @@ rec {
     containerdSha256 = "1isi1wgq61b4l0lxy1d8n6dnmcb8s5ihn2yqjb6525y3dj5c5i1j";
     tiniRev = "fec3683b971d9c3ef73f284f176672c44b448662"; # v0.18.0
     tiniSha256 = "1h20i3wwlbd8x4jr2gz68hgklh0lb0jj7y5xk1wvr8y58fip1rdn";
+    buildxSupport = buildxSupport;
```

Comment:
This causes infinite recursion when `buildxSupport` is not supplied, seems like the following works:

```
inherit buildxSupport;
```